### PR TITLE
adding my lab

### PR DESCRIPTION
**Pull request by r1zhok**

### CODE REVIEW

File: Clossing.h [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    Clothing(string name, string description, string location, string color, int size, enum Type type) {
+        this->name = name;
+        this->description = description;
+        this->location = location;
+        this->color = color;
+        this->size = size;
+        this->type = type;
+
+        cout << "name = " << this->name << endl;
```
Comment (Vadum4567890) [inline Clossing.h:48]: need to create printClothing() method for code: 
        cout << "name = " << this->name << endl;
        cout << "description = " << this->description << endl;
        cout << "location = " << this->location << endl;
        cout << "color = " << this->color << endl;
        cout << "size = " << this->size << endl;

File: lab5.cpp [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#include "class Clossing.h"
+#include "class Wardrobe.h"
+
+using namespace std;
+
+int main()
+{
+    Clothing A("Gucci", "very nice textile", "Paris", "black", 50, Shirt),
```
Comment (Vadum4567890) [inline lab5.cpp]: bad convention to Clothing objects

File: lab5.cpp [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        B("stani za 40uah", "idk", "China", "white", 70, Jeans),
+        C("Moncler", "Sheeesh", "London", "white", 40, Jacket);
+
+    Wardrobe D;
+    D.Add_Clo(A);
+    D.Add_Clo(B);
+    D.Add_Clo(C);
+    D.goOut();
+    D.Sort();
+
```
Comment (Vadum4567890) [inline lab5.cpp:19]: bad convention to Wardrobe object and methods in class must have same case-style

File: Wardrobe.h [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+    vector<Clothing> Clo;
+
+public:
+
+    void Add_Clo(Clothing& a) {
+        Clo.push_back(a);
+    }
+    void goOut() {
+        int n = 0;
```
Comment (Vadum4567890) [inline Wardrobe.h]: What is variable n? Set a correct name

File: Wardrobe.h [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+using namespace std;
+
+class Wardrobe {
+
+    friend Clothing;
+
+private:
+
+    vector<Clothing> Clo;
```
Comment (Vadum4567890) [inline Wardrobe.h]: Change Clo object to Clothing

**Review by Vadum4567890 — CHANGES_REQUESTED** [review]
change code

File: Clossing.h [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+using namespace std;
+
+enum Type { Shirt, Jeans, Jacket };
+string eType[] = { "Shirt", "Jeans", "Jacket" };
+
+class Clothing {
+
+public:
+
+    string name;
```
Comment (Vadum4567890) [inline Clossing.h:12]: set Clothing data to private: 
    string name;
    string description;
    string location;
    string color;
    int size;
    Type type;

**Review by Vadum4567890 — CHANGES_REQUESTED** [review]

File: lab5.cpp [hunk 7]
```diff
@@ -0,0 +1,22 @@
+#include "class Clossing.h"
+#include "class Wardrobe.h"
+
+using namespace std;
+
+int main()
+{
+    Clothing Shirt("Gucci", "very nice textile", "Paris", "black", 50, Shirt),
```
Comment (zveres-ss) [inline lab5.cpp:8]: variable names should start from lowercase letter

**Review by zveres-ss — CHANGES_REQUESTED** [review]

**Pull request merged**